openedx/features/ucsd_features/templates/logs_not_sent_email.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base_email.html' %}
+{% block body %}
+    <p>{{ body }}</p>
+    <br>
+
+    {% for info_dict in addition_info.items %}
+        <p><b>{{info_dict.0}}</b>: {{info_dict.1}}</p>
+        <br>
+    {% endfor %}
+{% endblock body %}
```

Comment:
why we have this file in this PR ?

Reply:
Actually This PR contains all those changes which are required to send events to Kafka.